### PR TITLE
Feat(Star War):Setup folder structure

### PR DESCRIPTION
**Pull request by Abhishek-Raj-AT**
Summary:
 setup folder Structure, Routing and redux toolkit
Change log:
I have done setup the folder structure, Routing and  redux toolkit

Test cases:
- [ ] Check folder structure name
- [ ] Check Routing code 
- [ ] Check redux toolkit code and I add the functionality of multiplication

### CODE REVIEW

File: src/Layout/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+import { Link, Outlet } from "react-router-dom";
+import { Strings } from "../resource/Strings";
+
+const Layout = () => {
```
Comment (bhavikg-dev) [inline src/Layout/Layout.tsx]: Ideally path should be `Layout/Page/index.tsx`

File: src/config/api.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const api ={
+    starChar: {
+        film: "/films/",
+        people: "/people",
+        planets: "/planets",
+        Species: "/species",
+        Vehicle: "/vehicle",
+        starShip: "/starShip",
```
Comment (bhavikg-dev) [inline src/config/api.tsx]: Remove initial `/` from all

File: src/config/config.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    environment: "development",
+  };
+  const isProd = false;
+  const deploymentType = isProd ? "production" : "development";
+  
+  if (deploymentType === "production") {
+    config = {
+      ...config,
+      siteUrl: "https://swapi.dev/api",
+      apiUrl: "https://swapi.dev/api",
```
Comment (bhavikg-dev) [inline src/config/config.tsx]: Put the `/` here

File: src/redux/FilmSlice/FilmType.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+export interface filmInformation {
```
Comment (bhavikg-dev) [inline src/redux/FilmSlice/FilmType.tsx]: Film

File: src/redux/FilmSlice/FilmType.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  skin_color: string;
+  eye_color: string;
+  birth_year: string;
+  gender: string;
+  homeworld: string;
+  films: string[];
+  species: string[];
+}
+export interface initialFilmList {
+  list: filmInformation[];
```
Comment (bhavikg-dev) [inline src/redux/FilmSlice/FilmType.tsx]: Film[]

File: src/redux/FilmSlice/FilmType.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  hair_color: string;
+  skin_color: string;
+  eye_color: string;
+  birth_year: string;
+  gender: string;
+  homeworld: string;
+  films: string[];
+  species: string[];
+}
+export interface initialFilmList {
```
Comment (bhavikg-dev) [inline src/redux/FilmSlice/FilmType.tsx]: Write. proper names here

File: src/redux/store.tsx [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import { configureStore } from "@reduxjs/toolkit";
+import { filmReducer } from "./FilmSlice/FilmSlice";
```
Comment (bhavikg-dev) [inline src/redux/store.tsx]: Fix here

File: src/resource/Strings.tsx [hunk 8]
```diff
@@ -0,0 +1,12 @@
+export const Strings = {
+  home: "Home",
+  films: "Films",
+  starShips: "StarShips",
+  people: "People",
+  vehicles: "Vehicles",
+  species: "Species",
+  planets: "Planets",
+  we_can_not_find_that_page: "We can not find that page",
```
Comment (bhavikg-dev) [inline src/resource/Strings.tsx]: Fix this

**Review by bhavikg-dev — CHANGES_REQUESTED** [review]

File: src/Layout/Layout.tsx (hunk 1 above)
Comment (Abhishek-Raj-AT) [reply src/Layout/Layout.tsx]: I changed the name here

File: src/config/api.tsx (hunk 2 above)
Comment (Abhishek-Raj-AT) [reply src/config/api.tsx]: I removed initial ` / ` from all

File: src/redux/FilmSlice/FilmType.tsx (hunk 6 above)
Comment (Abhishek-Raj-AT) [reply src/redux/FilmSlice/FilmType.tsx]: I changed the name here like FilmList

File: src/redux/store.tsx (hunk 7 above)
Comment (Abhishek-Raj-AT) [reply src/redux/store.tsx]: I solved this issue

File: src/resource/Strings.tsx (hunk 8 above)
Comment (Abhishek-Raj-AT) [reply src/resource/Strings.tsx]: I removed _ here

File: src/redux/FilmSlice/FilmType.tsx (hunk 5 above)
Comment (Abhishek-Raj-AT) [reply src/redux/FilmSlice/FilmType.tsx]: I also changed here

File: src/redux/FilmSlice/FilmType.tsx (hunk 4 above)
Comment (Abhishek-Raj-AT) [reply src/redux/FilmSlice/FilmType.tsx]: I changed the name filmInformation to Film

File: src/config/config.tsx (hunk 3 above)
Comment (Abhishek-Raj-AT) [reply src/config/config.tsx]: I put the / here

File: src/config/api.tsx [hunk 9]
```diff
@@ -1,11 +1,11 @@
...
-        planets: "/planets",
-        Species: "/species",
-        Vehicle: "/vehicle",
-        starShip: "/starShip",
+        film: "films/",
+        people: "people",
+        planets: "planets",
+        Species: "species",
+        Vehicle: "vehicle",
+        starShip: "starShip",
```
Comment (bhavikg-dev) [inline src/config/api.tsx]: Why / is not there same like `film` ?

**Review by bhavikg-dev — CHANGES_REQUESTED** [review]

**Review by bhavikg-dev — APPROVED** [review]

**Pull request merged**